versions.json

```diff
@@ -1,15 +1,13 @@
 {
-    "device": "1.17.0",
-    "service": "1.16.2",
-    "deps": "0.8.1",
-    "securityProvider": "1.2.0",
-    "tpmEmulator": "1.1.1",
-    "tpmHsm": "1.1.1",
-    "diceEmulator": "1.1.1",
-    "dice": "1.1.1",
-    "x509": "1.1.2",
-    "provisioning": "1.7.0",
-    "provisioningDevice": "1.6.0",
-    "provisioningService": "1.5.0",
-    "java": "0.24.0"
```

Comment:
What would  "java": "0.24.0", "provisioning": "1.7.0", follow incase of bump version only on preview? For consistency I suggest to add these as well to versions.json on preview branch

Reply:
`"java": "0.24.0", "provisioning": "1.7.0"` is parent version. They are not been used and shouldn't be changed. I'm not sure what they were used for but it should always be `1.0.0` to me.

Reply:
David is correct. The "java" and "provisioning" versions belonged to pom files that did not get published as artifacts. There was no reason for us to ever care what version they were, much less to bump them.